src/main/java/de/tum/in/www1/artemis/service/connectors/BitbucketBambooUpdateService.java

```diff
@@ -0,0 +1,114 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import com.appfire.bamboo.cli.BambooClient;
+import com.appfire.bitbucket.cli.BitbucketClient;
+import com.appfire.common.cli.CliClient;
+
+import de.tum.in.www1.artemis.config.Constants;
+import de.tum.in.www1.artemis.exception.BambooException;
+import de.tum.in.www1.artemis.service.connectors.bamboo.BambooBuildPlanUpdateProvider;
+
+@Service
+@Profile("bamboo & bitbucket")
```

Comment:
should this not be written like?
@Profile({ "bamboo", "bitbucket" })

Reply:
add comment

Reply:
Added a comment